assets/js/modules/analytics/components/dashboard/DashboardBounceRateWidget.js

```diff
@@ -55,9 +55,13 @@ function DashboardBounceRateWidget() {
 		const profileID = store.getProfileID();
 		const internalWebPropertyID = store.getInternalWebPropertyID();
 
+		const { compareStartDate, compareEndDate, startDate, endDate } = select( CORE_USER ).getDateRangeDates( { offsetDays: DATE_RANGE_OFFSET, compare: true, weekdayAlign: true } );
```

Comment:
This line is a bit long and would be better spread across a few. I thought we were spreading the date range dates into the args but it looks like we aren't doing that anymore.
Let's do something like this https://github.com/google/site-kit-wp/blob/8dc8de99f9225f62972c2bb82f67f6e3d39651ad/assets/js/modules/adsense/components/dashboard/AdSensePerformanceWidget.js#L42-L47

The args to `getDateRangeDates` should also be on separate lines though.

A good rule of thumb here is 1-2 keys destructured or shorthand key/value is okay. More than that destructuring, or when using key/values without the shorthand should probably be on separate lines.

```suggestion
		const {
			compareStartDate,
			compareEndDate,
			startDate,
			endDate
		} = select( CORE_USER ).getDateRangeDates( {
			offsetDays: DATE_RANGE_OFFSET,
			compare: true,
			weekdayAlign: true,
		} );
```